Library/Homebrew/missing_formula.rb

```diff
@@ -26,6 +26,10 @@ def blacklisted_reason(name)
           Minimal installation:
             brew cask install basictex
         EOS
+        when "asymptote" then <<~EOS
+          Asymptote is bundled with MacTeX. Install it via TeX Live Utility or:
+            tlmgr install asymptote
```

Comment:
`tlmgr` doesn't exist as a command for me. Can you state how to install MacTeX and/or TeX Live Utility here so it'll be output? Thanks!

Reply:
Clarified that `tlgmr` == TeX Live Utility, and it comes with MacTeX.